execution/rerun/rerun.go

```diff
@@ -142,7 +148,7 @@ func addFailedMetadata(res result.Result, args []string, add func(res result.Res
 }
 
 func writeFailedMeta(contents string) {
```

Comment:
Rename this to `writeLastRunInfo`, it is writing last run info and not failed meta.